### DISCUSSION TITLE
I want to use graph rag to extract some additional entities that I want to define manually

### DISCUSSION THREAD

**Discussion by streetstylecoder** [microsoft/graphrag#536] [Q&A]
, for example here when I am working on a businesss proposals , where there are hundreds of files , but I want to query "what was the average budget of proposals" or "what were the total number of proposals in software category"
which was always the problem with the naive rag because it can only return a specific number of business proposals , now that graph rag is here , I want to make use of it so that while embedding the data we can use the total summarised information of the set of files 

Can someone please help me out with the same ?

**Comment by akaashdash** [comment]
If you are using the cli, you can modify `entity_types` under `entity_extraction` in `settings.yaml`.
